### PR TITLE
Feature/sof 4399 - Stitching for ML Work

### PR DESCRIPTION
**Pull request by AcylSilane**

### CODE REVIEW

File: assets/python/espresso_xml_get_qpt_irr.j2.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 # ---------------------------------------------------------- #
 import json
 from xml.dom import minidom
+from __future__ import print_function
```
Comment (triplepoint) [inline assets/python/espresso_xml_get_qpt_irr.j2.py:11]: does this run as python2?

File: assets/python/ml/pyml:setup_variables_packages.j2.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 target_column_name = "target"
 
 # The type of ML problem being performed. Can be either "Regression", "Classification," or "Clustering."
```
Comment (triplepoint) [inline assets/python/ml/pyml:setup_variables_packages.j2.py:34]: stated allowed values don't match the case of the actual value

Comment (triplepoint) [reply assets/python/ml/pyml:setup_variables_packages.j2.py:34]: Also, why is this value hard-coded like this?  What's the behavior here?  I guess this file is the regression test asset? Then the filename should be changed to be similar to the clustering and classiffication files

**Review by triplepoint — CHANGES_REQUESTED** [review]

File: assets/python/espresso_xml_get_qpt_irr.j2.py (hunk 1 above)
Comment (AcylSilane) [reply assets/python/espresso_xml_get_qpt_irr.j2.py:11]: This file was still using the old Python2-style `print "spam"` syntax. I'm not sure if anything is still relying on it being Python2, so I went with the safer-route for back-compatibility

File: assets/python/ml/pyml:setup_variables_packages.j2.py (hunk 2 above)
Comment (AcylSilane) [reply assets/python/ml/pyml:setup_variables_packages.j2.py:34]: The regression test asset is in its own file for now - the test changes the value by itself. Initially, the intent here was to have one settings.py file that users could edit.

When translating this into the regression/clustering/classificaiton default workflows, there wasn't a good way to just change the value on-the-fly. I've made a task to do this more programatically here. We're going to be making it into an 'important setting' that just gets templated in: https://exabyte.atlassian.net/browse/SOF-4401

Comment (triplepoint) [reply assets/python/ml/pyml:setup_variables_packages.j2.py]: So why isn't this file named `assets/python/ml/pyml:setup_regression_variables_packages.j2.py` ?

**Review by triplepoint — APPROVED** [review]

**Pull request merged**